mob.go

```diff
@@ -1379,12 +1433,14 @@ func gitCommitHash() string {
 	return silentgitignorefailure("rev-parse", "HEAD")
 }
 
-func isGitInstalled() bool {
-	_, _, err := runCommandSilent("git", "--version")
+func gitVersion() (string, GitVersion) {
+	_, output, err := runCommandSilent("git", "--version")
 	if err != nil {
-		say.Debug("isGitInstalled encountered an error: " + err.Error())
+		say.Debug("gitVersion encountered an error: " + err.Error())
+		return "", GitVersion{}
 	}
-	return err == nil
+	versionString := strings.TrimSpace(output)
+	return versionString, parseGitVersion(versionString)
 }
```

Comment:
I'd rather have a method `gitVersion` that simply returns the git version string, and if necessary, the caller can then parse this to the `GitVersion` object. Makes it more readible, I think.